portfolio/src/main/java/com/google/sps/servlets/DeleteComments.java

```diff
@@ -0,0 +1,54 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.sps.servlets.DataServlet;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/** Servlet that returns some example content. */
+@WebServlet("/delete-data")
+public class DeleteComments extends HttpServlet {
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final String COMMENT_TABLE_NAME = "Comment";
+  private static final String TIMESTAMP_COLUMN_NAME = "submit_time";
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int numDeleteComments = 1;
```

Comment:
nit - this might be clearer as `defaultNumCommentsToDelete`? Likewise with the `inputNumDelete` below.

also consider moving the defaultNumCommentsToDelete to a constant.

Reply:
ah ok noted! if I move defaultNumCommentsToDelete to a constant, how can I change it later when I get the value?

Reply:
hmm it seems like we are talking about two different models. Currently, I'm using numDeleteComments as the baseline, and if the user updates the delete num value to a valid number, then I change that value to the user requested value. Do you think there is a better design here?

Reply:
This comment was more about the variable name. `numDeleteComments` reads more like "number of deleted comments" or "the number of deleted comments" (past tense). Feel free to be expressive with variable naming s.t. the name really reflects the purpose of the variable.